device-types/Cisco/C8300-6T

```diff
@@ -0,0 +1,37 @@
+---
+manufacturer: Cisco
+model: Catalyst 8300-6T
+slug: c8300-12x
+part_number: C8300-6T
```

Comment:
This is not correct.  There is a more appopriate model number, one of:

C8300-1N1S-4T2X
C8300-1N1S-6T
C8300-2N2S-6T
C8300-2N2S-4T2X